env0/resource_environment.go

```diff
@@ -0,0 +1,184 @@
+package env0
+
+import (
+	"context"
+	"github.com/env0/terraform-provider-env0/client"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+func resourceEnvironment() *schema.Resource {
+	return &schema.Resource{
+		CreateContext: resourceEnvironmentCreate,
+		ReadContext:   resourceEnvironmentRead,
+		UpdateContext: resourceEnvironmentUpdate,
+		DeleteContext: resourceEnvironmentDelete,
+
+		Importer: &schema.ResourceImporter{StateContext: resourceTeamImport},
```

Comment:
copy 🍝    `resourceTeamImport`

So are you implementing import now or on phase 2?

Reply:
903ab6d